core/ledger/kvledger/txmgmt/statedb/stateleveldb/persisted_value.proto

```diff
@@ -6,12 +6,12 @@ SPDX-License-Identifier: Apache-2.0
 
 syntax = "proto3";
 
-option go_package = "github.com/hyperledger/fabric/core/ledger/kvledger/txmgmt/statedb/stateleveldb/msgs";
+option go_package = "github.com/hyperledger/fabric/core/ledger/kvledger/txmgmt/statedb/stateleveldb";
 
-package msgs;
+package stateleveldb;
 
-message VersionedValueProto {
-    bytes version_bytes = 1;
+message PersistedValue {
```

Comment:
I don't find this change meaningful. "PersistedValue" is a very specific term that I may use for a variable when I assign it a value from a store. For a type, I don't find this term much meaningful.

Reply:
I see. Okay. Then, we need to pick between `VersionedValue` and `VersionedValueWithMetadata`. We cannot have the `Proto` in the variable name.  Or May be `DBValue` similar to `CacheValue` that we have.

Reply:
`DBValue` seems concise and meaningful to me

Reply:
Thanks, Manish. Will use that name then.